frontend/src/main/utils/riderUtils.js

```diff
@@ -0,0 +1,16 @@
+import { toast } from "react-toastify";
+
+export function onDeleteSuccess(message) {
+    console.log(message);
+    toast(message);
+}
+
+export function cellToAxiosParamsDelete(cell) {
```

Comment:
I think these are ride utils, not rider utils.

Reply:
commit [007145b](https://github.com/ucsb-cs156-s23/proj-gauchoride-s23-5pm-4/pull/32/commits/007145b7af202a2285f77a7ca76d7e8f4b67990f) should fix this